scripts/vim-patch.sh

```diff
@@ -392,7 +401,8 @@ list_missing_vimpatches() {
       continue
     fi
 
-    if vim_tag="$(git -C "${VIM_SOURCE_DIR}" describe --tags --exact-match "${vim_commit}" 2>/dev/null)"; then
+    vim_tag="${vim_commit_tags[$vim_commit]-}"
```

Comment:
syntax question: should there be a colon before the hyphen?
```suggestion
    vim_tag="${vim_commit_tags[$vim_commit]:-}"
```

Reply:
> ${parameter-default} and ${parameter:-default} are almost equivalent. The extra : makes a difference only when parameter has been declared, but is null.

(via https://www.tldp.org/LDP/abs/html/parameter-substitution.html)

Reply:
It makes me squirm, but ok.

Reply:
So you would prefer ":-"?  IIRC I've used the same as above, but that might have come from me already.